CHANGELOG.md

```diff
@@ -30,6 +30,11 @@
   ([PR #1009](https://github.com/jasmin-lang/jasmin/pull/1009);
   fixes [#1008](https://github.com/jasmin-lang/jasmin/issues/1008)).
 
+- Add support for x86 `VBLENDVPS` and `VBLENDVPD` instructions, through the new
+  intrinsics `#BLENDV` which also maps to the `VPBLENDVB` instruction;
```

Comment:
BLENDV seems to be the name of a x86 instruction working on floating point (BLENDVPD, BLENDVPS). Maybe VBLENDV will be a better name

Reply:
Looking more, I am not sure of what I say

Reply:
In order to pick a name, I sought inspiration in the [Intel® Intrinsics Guide](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=blendv).

Reply:
Looks good.